.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic",
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
Please add an entry to the .gitignore file making sure no files from .vscode folder are accidentally added to the commit